projects/spanner_emulator/project.yaml

```diff
@@ -0,0 +1,8 @@
+homepage: "https://github.com/googleinterns/cloud-spanner-emulator-fuzzing"
+language: c++
+primary_contact: "evmaus@google.com"
+auto_ccs:
+  - "snehashah@google.com"
+  - "volfson@google.com"
+sanitizers:
+ - address
```

Comment:
Why only address?

Reply:
I believe we were running into build issues when including the undefined sanitizer. We will be taking a closer look at which sanitizers to use later but our priority for now is to get the emulator fuzzer integrated with OSS-Fuzz.

Reply:
We should probably add memory as an experimental sanitizer as well.

We have run into issues with UBSAN compiling one of the direct dependencies, so I do think we want to skip that one for now.

Reply:
Memory has now been included as a sanitizer.